src/main/java/spotify/api/enums/GrantType.java

```diff
@@ -1,12 +1,26 @@
 package spotify.api.enums;
 
+/**
+ * This class contains and sets the grant type. This is the type of user authorization being requested.
```

Comment:
`This class contains all possible grant type values for authorizing` is better in my opinion.